.github/workflows/spelling3.yml

```diff
@@ -62,61 +83,48 @@ jobs:
     outputs:
       followup: ${{ steps.spelling.outputs.followup }}
     runs-on: ubuntu-latest
-    if: "contains(github.event_name, 'pull_request') || github.event_name == 'push'"
+    if: ${{ contains(github.event_name, 'pull_request') || github.event_name == 'push' }}
     concurrency:
       group: spelling-${{ github.event.pull_request.number || github.ref }}
       # note: If you use only_check_changed_files, you do not want cancel-in-progress
       cancel-in-progress: true
     steps:
       - name: check-spelling
         id: spelling
-        uses: check-spelling/check-spelling@v0.0.21
+        uses: check-spelling/check-spelling@v0.0.22
         with:
-          suppress_push_for_open_pull_request: 1
+          suppress_push_for_open_pull_request: ${{ github.actor != 'dependabot[bot]' && 1 }}
           checkout: true
           check_file_names: 1
-          spell_check_this: check-spelling/spell-check-this@prerelease
           post_comment: 0
           use_magic_file: 1
+          warnings: bad-regex,binary-file,deprecated-feature,ignored-expect-variant,large-file,limited-references,no-newline-at-eof,noisy-file,non-alpha-in-dictionary,token-is-substring,unexpected-line-ending,whitespace-in-dictionary,minified-file,unsupported-configuration,no-files-to-check
           extra_dictionary_limit: 10
-          extra_dictionaries:
+          extra_dictionaries: |
             cspell:cpp/src/compiler-msvc.txt
-            cspell:cpp/src/cpp.txt
             cspell:cpp/src/stdlib-c.txt
             cspell:cpp/src/stdlib-cpp.txt
             cspell:filetypes/filetypes.txt
-            cspell:fullstack/fullstack.txt
-            cspell:powershell/powershell.txt
-            cspell:software-terms/src/software-terms.txt
-
-  comment-push:
-    name: Report (Push)
-    # If your workflow isn't running on push, you can remove this job
-    runs-on: ubuntu-latest
-    needs: spelling
-    permissions:
-      contents: write
```

Comment:
I'm removing this job from the flow.

In the long term, I'm nudging people to rely on the step summary (which happens automatically) over adding comments, but in the near term, people can rely on comments when they make a pull request.